pkgs/os-specific/darwin/apple-sdk-12.3/libs/Xplugin.nix

```diff
@@ -0,0 +1,27 @@
+{
+  frameworks,
+  darwin-stubs,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
```

Comment:
```suggestion
stdenvNoCC.mkDerivation {
```

Reply:
Done. I’ve also sorted the attributes consistently between derivations. (I think I got them all.)

Marking this and the others resolved.